### ISSUE TITLE
Toggleable y-axis mouse support

### ISSUE THREAD

**Issue by sneakernets** [viti95/FastDoom#79]
Consider re-adding code that was removed from [this commit](https://github.com/viti95/FastDoom/commit/c53cbe395b7ce0a2423a40425d03068868f7f81b) to be enabled only through a command-line parameter.

 I propose `-mousey`

**Comment by RamonUnch** [comment]
I agree, this would be nice, I am used to play with vertical mouse movement and I think it would be even better to have a key to toggle vertical mouse movement state like it can be done with crispy-doom's `key_togglenovert` option.
